deploy/upgrades/blocking-mechanisms/cgu-a.yaml

```diff
@@ -6,14 +6,20 @@ metadata:
   annotations:
     cluster-group-upgrades-operator/name-suffix: kuttl  
 spec:
+  actions:
+    afterCompletion:
+      deleteClusterLabels:
+        upgrade: ""
```

Comment:
Not sure how to verify if the finish label is only added in the last step. For the deletion though, we could verify that  it exists on the clusters until the end?